### ISSUE TITLE
[GTK] ConfigureEvent can't received

### ISSUE THREAD

**Issue by lld1995** [unoplatform/uno#13971]
### Current behavior

i can't receive this event when i move window

### Expected behavior

receive this event when i move window

### How to reproduce it (as minimally and precisely as possible)

`GtkHost.Window.AddEvents((int)EventMask.StructureMask);
GtkHost.Window.ConfigureEvent += Window_ConfigureEvent;`

### Works on UWP/WinUI

None

### NuGet package version(s)

4.9.49

### Affected platforms

Skia (GTK on Linux/macOS/Windows)

### IDE

Visual Studio 2022

**Comment by lld1995** [comment]
RequestedEvents in GtkCoreWindowExtension doesn't contain Gdk.EventMask.StructureMask,but gtk required

**Comment by lld1995** [comment]
i guess it should be set before window create

**Comment by Youssef1313** [comment]
https://github.com/unoplatform/uno/blob/22c8e1d0288f2c0a8c2820c4a9984f27f5cfec66/src/Uno.UI.Runtime.Skia.Gtk/Helpers/Dpi/DpiHelper.cs#L33

https://github.com/unoplatform/uno/blob/22c8e1d0288f2c0a8c2820c4a9984f27f5cfec66/src/Uno.UI.Runtime.Skia.Gtk/Helpers/Dpi/DpiHelper.cs#L48

@jeromelaban This might be the root cause of the issue you were seeing on Gtk?

**Comment by Youssef1313** [comment]
The interesting part here is that in your repro steps you already added `EventMask.StructureMask` so it's unclear why the event is not being raised.

**Comment by lld1995** [comment]
> The interesting part here is that in your repro steps you already added `EventMask.StructureMask` so it's unclear why the event is not being raised.

i think it may set mask before the window create?

**Comment by Youssef1313** [comment]
Well, I think StructureMask belongs to the `Gdk` window, i.e, `GtkHost.Window.Window` (note the second `Window` access). But I'm seeing that `GtkHost.Window.Window.Events` already has `StructureMask`

Per the [documentation](https://docs.gtk.org/gtk3/signal.Widget.configure-event.html), it's supposed to work. I can't spot why it doesn't.

**Comment by lld1995** [comment]
> Well, I think StructureMask belongs to the `Gdk` window, i.e, `GtkHost.Window.Window` (note the second `Window` access). But I'm seeing that `GtkHost.Window.Window.Events` already has `StructureMask`
> 
> Per the [documentation](https://docs.gtk.org/gtk3/signal.Widget.configure-event.html), it's supposed to work. I can't spot why it doesn't.

No,i don't think so,because it triggers on WndProc belongs to first window,the doc mean if you set GtkHost.Window,all window you create will have the mask.

**Comment by jeromelaban** [comment]
Interesting findings. It does not ring a bell, but definitely @MartinZikmund will be interested for the multi-window work.

**Comment by MartinZikmund** [comment]
As GTK is no longer supported and was replaced by X11 backend, closing this issue
